src/status_im/ui/screens/discover/subs.cljs

```diff
@@ -49,3 +49,12 @@
           current-tag (:current-tag db)
           tags        (:discover-search-tags db)]
       (get-discoveries-by-tags discoveries current-tag tags))))
+
+(reg-sub :get-all-dapps
```

Comment:
https://github.com/status-im/status-react/blob/883b702fcba7d94b93828ccfaa6d6053100fe52d/src/status_im/ui/screens/contacts/subs.cljs#L35
```
(reg-sub
  :all-added-dapp-contacts
  :<- [:all-added-contacts]
  (fn [contacts]
    (filter #(true? (:dapp? %)) contacts)))
```

Reply:
this is just dapps in one group `[:group/contact-groups "dapps" :contacts]` , user can remove dapps from this group or remove this group at all

Reply:
We decided to start with this, can fix later

Reply:
ok

Reply:
Yeah, Jarrad will have to curate the list again anyway, so no point tweaking it now.